mlir/include/mlir/Conversion/LLVMCommon/VectorPattern.h

```diff
@@ -98,6 +99,15 @@ class VectorConvertToLLVMPattern : public ConvertOpToLLVMPattern<SourceOp> {
     static_assert(
         std::is_base_of<OpTrait::OneResult<SourceOp>, SourceOp>::value,
         "expected single result op");
+
+    // Check if the operation is remove the fastMathAttr on ExtFOp  / TruncFOp.
+    if (isa<arith::ExtFOp>(op.getOperation()) ||
+        isa<arith::TruncFOp>(op.getOperation())) {
+      if (op->hasAttr("fastmath")) {
+        op->removeAttr("fastmath");
+      }
+    }
+
```

Comment:
Is it OK to passdown the extf / truncf fastmathflag `contract` to llvm. Do we need such a remove attr logic here? @krzysz00 @kuhar